src/main/java/org/osiam/addons/self_administration/controller/ExceptionController.java

```diff
@@ -0,0 +1,30 @@
+package org.osiam.addons.self_administration.controller;
+
+import org.springframework.boot.autoconfigure.web.ErrorController;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.servlet.ModelAndView;
+
+import javax.annotation.PostConstruct;
+
+@Controller
+public class ExceptionController implements ErrorController {
+
+    private ModelAndView modelAndView = new ModelAndView("self_administration_error");
+
+    @PostConstruct
+    public void createErrorPage() {
+        modelAndView = new ModelAndView("self_administration_error");
```

Comment:
why creating a new `ModelAndView` again? It already gets initialized at the field declaration.